OmsAgent/Fairfax/omsagent_ff.py

```diff
@@ -731,7 +731,7 @@ def log_and_exit(hutil, operation, exit_code = 1, message = ''):
     if hutil is not None:
         hutil.do_exit(exit_code, operation, exit_status, str(exit_code), message)
     else:
-        update_status_file(operation, exit_code, exit_status, message)
+        update_status_file(operation, str(exit_code), exit_status, message)
```

Comment:
This change is from commit https://github.com/Azure/azure-linux-extensions/pull/393/commits/c90649a2cfa93db092bf9bdfb61767e39f1cff07

This change is already included in the current public version of the extension, but it hasn't been migrated to the Azure code base yet. It's a small change, and it was quicker to include it in this PR.